app/questionnaire/navigator.py

```diff
@@ -101,6 +101,11 @@ def __init__(self, survey_json, metadata=None, answer_store=None):
         self.metadata = metadata or {}
         self.survey_json = survey_json
 
+        self.preceeding_path = []
```

Comment:
I think we need to be able to distinguish which pages are part of the question flow, and which sit outside of it(introduction, summary). For this it could be clearer for them to remain implemented outside of the groups and blocks. For now, this change makes sense, and we can review the wider options in the new year.